worker/include/Channel/Request.hpp

```diff
@@ -76,15 +76,20 @@ namespace Channel
 		void TypeError(const char* reason = nullptr);
 
 	public:
-		// Passed by argument.
-		Channel::UnixStreamSocket* channel{ nullptr };
 		uint32_t id{ 0u };
 		std::string method;
 		MethodId methodId;
 		json internal;
 		json data;
 		// Others.
 		bool replied{ false };
+
+	public:
```

Comment:
Move static methods and members above, below the `UnixStreamSocket` constructor.

Reply:
`UnixStreamSocket` constructor ? I didn't get it.  change like this?
```
class Request
	{
	public:
		static void ClassInit(Channel::UnixStreamSocket* channel);

	public:
		// Passed by argument.
		static Channel::UnixStreamSocket* channel;
		//....
	}
```